.github/workflows/ci.yml

```diff
@@ -21,6 +21,11 @@ jobs:
       - name: Install Gems
         run: bundle install
 
+      - name: Verify cocoapods
+        run: |
+          sudo gem install cocoapods
+          pod lib lint
```

Comment:
What's the purpose of this step?

Reply:
This validates that the pods are fine. For instance if we add a file that causes issues on one of the supported architectures.

Reply:
I see, that makes sense